nodejs/README.adoc

```diff
@@ -0,0 +1,341 @@
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As a developer of applications and services, you can connect Node.js applications to Kafka instances in {product-long}. https://nodejs.org/en/about/[Node.js^] is a server-side JavaScript runtime that is designed to build scalable network applications. Node.js provides an I/O model that is based on events and non-blocking operations, which enables efficient applications.
+
+.Prerequisites
+ifndef::community[]
+* You have a Red Hat account.
+endif::[]
+* You have a Kafka instance in {product} and the instance is in the *Ready* state. To learn how to create a Kafka instance, see link:{base-url}{getting-started-url}[Getting started with {product-long}].
+* https://github.com/git-guides/[Git^] is installed.
+* You have an IDE such as https://www.jetbrains.com/idea/download/[IntelliJ IDEA^], https://www.eclipse.org/downloads/[Eclipse^], or https://code.visualstudio.com/Download[VSCode^].
+* https://nodejs.org/en/download/[Node.js 14.x^] is installed.
+
+[NOTE]
+====
+The example Node.js application in this quick start uses a https://kafka.js.org/[KafkaJS^] client by default. If you want to use a https://github.com/blizzard/node-rdkafka[node-rdkafka^] client, you must install Docker to run a specified container image and configure a development environment. To learn more, see the https://github.com/nodeshift-starters/reactive-example/blob/main/README.md[documentation] for the Node.js application.
+====
+
+// Condition out QS-only content so that it doesn't appear in docs.
+// All QS anchor IDs must be in this alternate anchor ID format `[#anchor-id]` because the ascii splitter relies on the other format `[id="anchor-id"]` to generate module files.
+ifdef::qs[]
+[#description]
+Learn how to use Node.js applications to produce and consume messages using a Kafka instance in {product-long}.
+
+[#introduction]
+Welcome to the quick start for {product-long} with Node.js. In this quick start, you'll learn how to use https://nodejs.org/en/about/[Node.js^] to produce messages to and consume messages from your Kafka instances in {product}.
+endif::[]
+
+
+[id="proc-importing-nodejs-sample-code_{context}"]
+== Importing the Node.js sample code
+
+For this quick start, you'll use sample code from the Nodeshift Application Starters https://github.com/nodeshift-starters/reactive-example[reactive-example^] repository in GitHub. After you understand the concepts and tasks in this quick start, you can use your own Node.js applications with {product} in the same way.
+
+.Procedure
+. On your computer, open a command-line window.
+. On the command line, clone the Nodeshift Application Starters https://github.com/nodeshift-starters/reactive-example[reactive-example^] repository from GitHub.
```

Comment:
```suggestion
. On the command line, clone the Nodeshift Application Starter https://github.com/nodeshift-starters/reactive-example[reactive-example^] repository from GitHub.
```

Reply:
No strong opinion. I just found it odd when reading.
That name reflects organization. What we cloning is repository which is instance of single starter.

Reply:
Yes, sorry, _organization_ was the word I was looking for - not project. Regardless, I think we are right to name it as it appears.